src/App.tsx

```diff
@@ -15,48 +16,101 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Name = 'name',
+  Length = 'length',
+  Default = '',
+}
+
+function sortGoods(
+  goods: string[],
+  sortType: SortType,
+  isReversed: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortType) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortType) {
+        case SortType.Length:
+          return good1.length - good2.length;
+
+        case SortType.Name:
+          return good1.localeCompare(good2);
+
+        default:
+          return 0;
```

Comment:
Would be great to move switch...case out of the sort's callback to avoid extra `sort` calls if it's a default case